exe/driver.cc.in

```diff
@@ -42,6 +42,21 @@ void help(po::options_description& desc)
           "Displays its own version string.", desc) << endl;
 }
 
+void conflicting_options(po::variables_map const& vm, string const& opt1, string const& opt2)
+{
+    if (vm.count(opt1) && !vm[opt1].defaulted() && vm.count(opt2) && !vm[opt2].defaulted()) {
+        throw po::error(opt1 + " and " + opt2 + " options conflict: please specify only one.");
+    }
+}
+
+template<typename... TOpts>
+void conflicting_options(po::variables_map const& vm, string const& opt1, string const& opt2, TOpts && ...opts)
+{
```

Comment:
Writing a full permutation comparison with a vector might actually be more complex. Want to propose an implementation?